contracts/RiskManagerV1.sol

```diff
@@ -517,6 +535,19 @@ contract RiskManagerV1 is IRiskManagerV1, Auctioneer, Ownable {
         signerBondsSwapStrategyInitiated = 0;
     }
 
+    /// TODO: add description
+    ///      - do we need a delay "onlyAfterGovernanceDelay" for updating council
+    ///        multisig?
```

Comment:
Could be worth it, yes. But this logic can be a part of the risk manager.

Reply:
Done